lib/puppet/type/cisco_aaa_group_tacacs.rb

```diff
@@ -0,0 +1,143 @@
+# Manages configuration for a TACACS+ server group.
+#
+# October 2015
+#
+# Copyright (c) 2013-2015 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+Puppet::Type.newtype(:cisco_aaa_group_tacacs) do
+  @doc = "Manages configuration for a TACACS+ server group
+
+  cisco_aaa_group_tacacs {\"<group>\":
+    ..attributes..
+  }
+
+  <group> is the name of the TACACS+ server group resource.
+
+  Example:
+    cisco_aaa_group_tacacs {\"testgroup1\":
+      ensure           => present,
+      deadtime         => 30,
+      vrf_name         => \"blue\",
```

Comment:
group is the namevar, should I include a duplicate entry for it with the properties?